web/cypress/integration/morecast-page.spec.ts

```diff
@@ -63,29 +81,59 @@ describe('MoreCast Page', () => {
     cy.url().should('contain', `${timeOfInterestQueryKey}=${timeOfInterest}`)
   })
 
+  describe('When loading a single station from url', () => {
+    beforeEach(() => {
+      interceptData()
+
+      cy.visit(`${MORECAST_ROUTE}?codes=${stationCode}`)
+      cy.wait('@getStations')
+    })
+
+    it('should load with an observation table', () => {
+      cy.getByTestId(`observations-table-${stationCode}`)
+        .find('tbody > tr')
+        .should('have.length', numOfObservations)
+
+      // expect the sidepanel to be partially expanded (we compare the calculated width, and expect
+      // it to match the width of our browser window)
+      // NOTE: Using an exact pixel width is an issue, as the pixel width can be slightly different
+      // from platform to platform.
+      cy.getByTestId('expandable-container-content').should('have.css', 'width', '787px')
```

Comment:
@andrea-williams @conbrad any better ideas here? Using the width to test is an issue, since it changes from environment to environment. 787 works on github.